README.md

```diff
@@ -91,6 +101,20 @@ network_connections:
 network_allow_restart: yes
 ```
 
+```yaml
+network_provider: nm
+network_state:
+  interfaces:
+    - name: eth0
+    #...
+  routes:
+    config:
+      #...
+  dns-resolver:
+    config:
+      #...
+```
```

Comment:
As this is an example.. should we use complete states instead of using `#...`? What do you think?

Reply:
If you see the few lines before it, 
```yaml
network_provider: nm
network_connections:
  - name: eth0
    #...
network_allow_restart: yes
```
I just followed the format above, I also added the complete state examples  below.